### ISSUE TITLE
[GSSoC25] Add ARIA labels

### ISSUE THREAD

**Issue by vector1505** [DhruvJohri/Netflix-Clone#90]
### 📝 Title / Summary

Adding aria-labels to all the tags

### 🔍 What's the idea or issue?

Opening a new issue since this clashed with the earlier one

The site currently lacks aria-labels, making it not very accessible to screen readers

### ⚙️ How will you solve it?

1. I have forked the repo and started work
2. Adding aria-labels to most if not all elements

### ✅ GSSoC '25 Checklist

- [x] I have searched and confirmed this issue is not already open.
- [x] I am a registered participant of GSSoC 2025.
- [x] I have starred the repository [Star_here](https://github.com/DhruvJohri/Netflix-Clone)
- [x] I agree to follow the project's [Code of Conduct](https://github.com/DhruvJohri/Netflix-Clone/blob/main/CODE_OF_CONDUCT.md).
- [x] I follow [@DhruvJohri_ on X](https://x.com/DhruvJohri_) for updates and announcements.

**Comment by Afshan0608** [comment]
Hi! I'm Afshan, a GSSoC'25 contributor. I'd love to work on this issue as my first contribution. Could you please assign it to me? 😊
